.github/workflows/release_to_docker.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         with:
           context: ./
           file: ./Dockerfile
-          platforms: linux/amd64
+          platforms: linux/amd64, linux/arm64
```

Comment:
I'm not sure this is the only change needed for this to work. If you follow the instructions in the link posted in the description above, you'll notice the chapter describing QEMU setup, that the buildx step possibly needs to succeed in the runner. (See https://docs.docker.com/build/building/multi-platform/ for general info…) — My guess is that at least `docker/setup-qemu-action` to register the emulation types would be needed…?